configuration/things.md

```diff
@@ -64,15 +64,19 @@ See the [configuration tutorial]({{base}}/tutorials/beginner/configuration.html)
 Things can also be defined manually by creating `.things` configuration text files.
 These files are stored in `$OPENHAB_CONF/things`.
 
-Benefits of defining Things, Items and other aspects of openHAB in configuration text files are, that they are statically defined, unambiguous, flexible and easy to backup and restore.
-The main downsides of configuration files are the effort needed to compose them and the probability for typing errors.
-
 The syntax for `.things` files is defined as follows (parts in `<..>` are required):
 
 ```xtend
 Thing <binding_id>:<type_id>:<thing_id> "Label" @ "Location" [ <parameters> ]
 ```
 
+The first keyword defines whether the entry is a bridge or a thing. The next statement defines the UID of the thing which contains of the following three segments: binding id, thing type id, thing id. So the first two segments must match to thing type supported by a binding (e.g. `network:device` or `astro:moon`), whereas the thing id can be freely defined. Optionally, you may provide a label in order to recognize it easily, otherwise the default label from the thing type will be displayed.
```

Comment:
Could please put each sentence on a line by itself? This makes reviewing further changes much easier, and the website builder takes care of rendering proper paragraphs.

Reply:
Done.